q2_humann3/plugin_setup.py

```diff
@@ -86,20 +88,70 @@
         "metaphlan_stat_q": "Quantile value for the robust average",
     },
     output_descriptions={
-        "genefamilies": ("This file details the abundance of each gene family"
-                         " in the community."),
-        "pathcoverage": ("Pathway coverage provides an alternative description"
-                         " of the presence (1) and absence (0) of pathways in"
-                         " a community, independent of their quantitative"
-                         " abundance."),
-        "pathabundance": ("This file details the abundance of each pathway in"
-                          " the community as a function of the abundances of"
-                          " the pathway's component reactions, with each"
-                          " reaction's abundance computed as the sum over"
-                          " abundances of genes catalyzing the reaction."),
-        "taxonomy": ("Taxonomic profile of microbial community of samples,"
-                     " generated using clade-specific marker genes."),
+        "genefamilies": (
+            "This file details the abundance of each gene family" " in the community."
+        ),
+        "pathcoverage": (
+            "Pathway coverage provides an alternative description"
+            " of the presence (1) and absence (0) of pathways in"
+            " a community, independent of their quantitative"
+            " abundance."
+        ),
+        "pathabundance": (
+            "This file details the abundance of each pathway in"
+            " the community as a function of the abundances of"
+            " the pathway's component reactions, with each"
+            " reaction's abundance computed as the sum over"
+            " abundances of genes catalyzing the reaction."
+        ),
+        "taxonomy": (
+            "Taxonomic profile of microbial community of samples,"
+            " generated using clade-specific marker genes."
+        ),
     },
     name="Characterize samples using HUMAnN3",
     description="Execute the HUMAnN3",
 )
+
+plugin.methods.register_function(
+    function=q2_humann3.rename_table,
+    inputs={"table": FeatureTable[Frequency | RelativeFrequency]},
+    parameters={
+        "name": Str  # type: ignore
+        % Choices(
+            {
+                "kegg-orthology",
+                "kegg-pathway",
+                "kegg-module",
+                "ec",
+                "metacyc-rxn",
+                "metacyc-pwy",
+                "pfam",
+                "eggnog",
+                "go",
+                "infogo1000",
+            }
+        ),
+        "simplify": Bool,
+    },
+    name="Rename Table",
+    outputs=[
+        ("renorm_table", FeatureTable[Frequency | RelativeFrequency]),  # type: ignore
```

Comment:
Just to clarify, does it return a "renorm_table" or "rename_table"?

Reply:
Rename, good catch